XenAdmin/Wizards/PatchingWizard/AutomatedUpdatesBasePage.cs

```diff
@@ -332,9 +333,32 @@ private void WorkerDoWork(object sender, DoWorkEventArgs doWorkEventArgs)
                         bgw.RunPlanAction(action, ref doWorkEventArgs);
                     }
 
+                    // Step 2.5: Rearrange DelayedActions

```

Comment:
2.5?? Where are 2.1-2.4?

Reply:
cause I was hesitated to change all the following number, in case the code needs to be reshuffled again, so just put 2.5 indicating this is between step 2 and 3. Sure I'll correct all of them later.

Reply:
In this case use 2.1 :) I got the impression there was more stuff inbetween that was left out of the PR